src/org/usfirst/frc/team2335/robot/GripPipeline.java

```diff
@@ -0,0 +1,228 @@
+package org.usfirst.frc.team2335.robot;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.opencv.core.Core;
+import org.opencv.core.CvType;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfInt;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.MatOfPoint2f;
+import org.opencv.core.Point;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.imgproc.Imgproc;
+
+import edu.wpi.first.wpilibj.vision.VisionPipeline;
+
+/**
+* GripPipeline class.
+*
+* <p>An OpenCV pipeline generated by GRIP.
+*
+* @author GRIP
+*/
+public class GripPipeline implements VisionPipeline {
```

Comment:
What is the GripPipeline class? Also VisionPipeline and other pipeline things used later.